packages/vite/src/node/config.ts

```diff
@@ -1720,6 +1720,8 @@ async function bundleConfigFile(
     format: isESM ? 'esm' : 'cjs',
     mainFields: ['main'],
     sourcemap: 'inline',
+    // the last slash is needed to make the path correct
+    sourceRoot: path.dirname(fileName) + '/',
```

Comment:
```suggestion
    sourceRoot: path.dirname(fileName) + path.sep,
```

Should this use `path.sep` since it could be backward slash now?

Reply:
It seems to work with `/`, but yeah, `path.sep` feels more correct. Updated 👍